BraintreeUIKit/Public/BTUIKAppearance.h

```diff
@@ -2,11 +2,17 @@
 
 @interface BTUIKAppearance : NSObject
 
+typedef NS_ENUM(NSInteger, BTUIKColorScheme) {
+    BTUIKColorSchemeLight,
+    BTUIKColorSchemeDark,
+    BTUIKColorSchemeDynamic API_AVAILABLE(ios(13.0))
+};
+
 /// Shared instance used by Form elements
 + (instancetype)sharedInstance;
 
-+ (void)darkTheme;
-+ (void)lightTheme;
++ (void)darkTheme DEPRECATED_MSG_ATTRIBUTE("Use colorScheme property instead");
++ (void)lightTheme DEPRECATED_MSG_ATTRIBUTE("Use colorScheme property instead");
```

Comment:
We think it makes sense to deprecate these class methods in favor of the colorScheme property. But we're open to not deprecating if anyone feels differently.